Keycloak Dev service does not start with OIDC client with 3.20 and 3.21

### Describe the bug

When creating a plain project and adding the quarkus-oidc-client extension,  the launch of a keycloak dev service fails. See the reproducer https://github.com/jelmew/quarkus_dev_service_broken 


### Expected behavior

It is expected that the test starts a container, prints hello world and shutdowns again.

### Actual behavior

Test times out  waiting for

```
Caused by: org.testcontainers.containers.ContainerLaunchException: Timed out waiting for log output matching '.*Keycloak.*started.*'
        at org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy.waitUntilReady(LogMessageWaitStrategy.java:47)
        at org.testcontainers.containers.wait.strategy.AbstractWaitStrategy.waitUntilReady(AbstractWaitStrategy.java:52)
        at org.testcontainers.containers.GenericContainer.waitUntilContainerStarted(GenericContainer.java:909)
        at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:492)
        ... 18 more
```

### How to Reproduce?

https://github.com/jelmew/quarkus_dev_service_broken

If debug logging is enabled, you see that the container never prints this log line at all.

### Output of `uname -a` or `ver`

Linux pop-os 6.12.10-76061203-generic #202412060638~1743109366~22.04~1fce33b SMP PREEMPT_DYNAMIC Thu M x86_64 x86_64 x86_64 GNU/Linux

### Output of `java -version`

openjdk version "23.0.2" 2025-01-21 OpenJDK Runtime Environment Temurin-23.0.2+7 (build 23.0.2+7) OpenJDK 64-Bit Server VM Temurin-23.0.2+7 (build 23.0.2+7, mixed mode, sharing)

### Quarkus version or git rev

3.20

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

Apache Maven 3.9.9 (8e8579a9e76f7d015ee5ec7bfcdc97d260186937)

### Additional information

_No response_

--- a/extensions/devservices/keycloak/src/main/java/io/quarkus/devservices/keycloak/KeycloakDevServicesConfig.java
+++ b/extensions/devservices/keycloak/src/main/java/io/quarkus/devservices/keycloak/KeycloakDevServicesConfig.java
@@ -206,9 +206,9 @@ public interface KeycloakDevServicesConfig {
     /**
      * Memory limit for Keycloak container, up to {@code Long.MAX_VALUE} bytes.
      * </p>
-     * If not specified, 750MiB is the default memory limit.
+     * If not specified, 1250MiB is the default memory limit.
      */
-    @WithDefault("750M")
+    @WithDefault("1250M")
     MemorySize containerMemoryLimit();
 
     /**

